docs/eventing/broker/kafka-broker/README.md

```diff
@@ -142,6 +144,10 @@ data:
         namespace: knative-eventing
 ```
 
+## Using externally managed topic
```

Comment:
What if we call this section "Bring your own topic" ? :thinking:

Reply:
fine with me